src/styles/index.scss

```diff
@@ -1,3 +1,147 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+h3,
+p {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.card {
+  max-width: 200px;
+  width: 100%;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__img {
+    width: 100%;
+    margin-bottom: 40px;
+  }
+  &__prod {
+    margin-bottom: 16px;
+  }
+  &__rewie {
```

Comment:
There is a typo in the class name '&__rewie'. It should be '&__review' to match the corrected class name in the HTML file.